### PR TITLE
fix warnings in integration test

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

**Review by pmaytak — APPROVED** [review]

File: tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class AcquireTokenForAppIntegrationTests
             else
             {
                 _output.WriteLine("Connection to Key Vault failed. No secret returned. ");
-                throw new ArgumentNullException(nameof(_ccaSecret), "No secret returned from Key Vault. ");
+                throw new ArgumentNullException(message: "No secret returned from Key Vault. ", null);
             }
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs:50]: I don't understand this fix?

**Review by jmprieur — COMMENTED** [review]
Maybe the null argument exception fix woukd need explanation?

File: tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs:50]: _ccaSecret is not a parameter, so i'm just returning the message instead. ParamName is specifically for a parameter that is null.

Comment (jmprieur) [reply tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs:50]: I see. Strickly speaking this probably shouldn't be an `ArgumentNullException`, then. But I don't have a better idea.

**Pull request merged**